packages/nexus-sdk/src/Archive/__tests__/index.spec.ts

```diff
@@ -39,6 +39,52 @@ describe('Archive', () => {
       expect(fetchMock.mock.calls[0][1].method).toEqual('GET');
     });
 
+    it('should call httpGet with the correct parseAs context property', async () => {
+      const mockHttpGet = jest.fn();
```

Comment:
I don't mind doing it this way, but it is not consistent with the rest of the tests where we directly use mockFetch...

Reply:
It allows us to access arguments which we pass when httpGet is called, otherwise it is tricky to see parseAs value

Reply:
I see 🧐 !